app/assets/stylesheets/components/_button.scss

```diff
@@ -37,3 +37,95 @@
   }
 }
 
+.btn-medium {
+  color: #3B3B3C;
+  border: 1px solid #999999;
+  padding: 10px 15px;
+  border-radius: 50px;
+  font-weight: lighter;
+  opacity: 0.5;
+  background-color: lightblue;
```

Comment:
Can we set the new $blue to the lightblue so we can change it all once if needed ?